djstripe/settings.py

```diff
@@ -14,6 +14,7 @@
 
 INVOICE_FROM_EMAIL = getattr(settings, "DJSTRIPE_INVOICE_FROM_EMAIL", "billing@example.com")
 PAYMENTS_PLANS = getattr(settings, "DJSTRIPE_PLANS", {})
+DJSTRIPE_PLAN_HIERARCHY = getattr(settings, "DJSTRIPE_HIERARCHY_PLANS", {})
```

Comment:
Sorry if I wasn't clear enough, but this should read:
``PLAN_HIERARCHY = getattr(settings, "DJSTRIPE_PLAN_HIERARCHY", {})``